content/quests/events/demevent_bog_vaccine.lua

```diff
@@ -17,129 +17,169 @@ QDEF:AddConvo()
             ]],
             OPT_ASK_INFESTATION = "Ask about the infestation",
             DIALOG_ASK_INFESTATION = [[
                 player:
-                    How did the bog parasites get in the Pearl?
+                    How did Squeaky Clean Pearl get this infestation?
+                agent:
+                    Well, with the new election system, we had signed a truce deal allowing most willing to vote into the Pearl.
+                    Unfortunately, that opened Hesh's flood gates for a breed of heretics we normally kept in Grout Bog.
+                    These, er, <i>people<\> brought their own ideas, and those ideas brought sickness to Hesh's Pious.
+                player:
+                    !question
+                    Is this infection as bad as you're doomsaying it to be?
                 agent:
-                    As part of the truce deal, every heretic is allowed in the Pearl apparently.
-                    And some boggers brought their filth into the Pearl.
-                    We plan to stop the infestation at its root before it becomes an epidemic.
+                    Hesh's wrath is harsh to those unfit for consumption.
+                    Should you be infected without proper treatment, your death will be slow, and your soul damned.
                 player:
-                    What are the consequences of catching the parasite?
+                    Let's focus on the harsh and painful death part first.
                 agent:
-                    It would be a disgrace to Hesh itself.
-                    And you will probably die horribly. But that is less important than the insult to Hesh.
+                    That's what most people care about more, short-sighted though it may be.
             ]],
             OPT_ASK_VACCINE = "Ask about the vaccine",
             DIALOG_ASK_VACCINE = [[
                 player:
-                    What is this vaccine you speak of?
+                    So what's in all those vials that you'd be stabbing me with?
                 agent:
-                    It is a preemptive measure to the bog infestation.
-                    Once you are innoculated, you are guaranteed to never get infected by the parasite.
-                    It is 100% reliable.
-                    Although it has a temporary side effect that won't be pleasant.
-                    But that is a small price to pay for salvation.
+                    Uncouthly put, {player}. I assure you, this is a completely safe preventative measure.
+                    You may feel some minor aches and I wouldn't reccommend you exert yourself.
+                    But that is just your body crafting it's shield of faith against this heretical poison. 
+                    You'll shrug away the pains quickly, and your soul will be prepared for Consumption by Hesh, as all who walk in the shallows deserve.
```

Comment:
Why is "Consumption" capitalized?

Reply:
I can't remember, I think I just flip flopped between Capitalized because it was a supposed religious event and not. I'll check the game when it mentions consumption and change it accordingly